circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/metastore/TunnellingMetaStoreClientSupplier.java

```diff
@@ -66,15 +66,16 @@ private static int getLocalPort() {
     }
   }
 
-  private static class HiveMetaStoreClientSupplier implements TunnelableSupplier<CloseableMetaStoreClient> {
+  @VisibleForTesting
+  static class HiveMetaStoreClientSupplier implements TunnelableSupplier<CloseableMetaStoreClient> {
     private final MetaStoreClientFactory metaStoreClientFactory;
-    private final HiveConf hiveConf;
     private final String name;
+    final @VisibleForTesting HiveConf hiveConf;
 
     private HiveMetaStoreClientSupplier(MetaStoreClientFactory metaStoreClientFactory, HiveConf hiveConf, String name) {
       this.metaStoreClientFactory = metaStoreClientFactory;
-      this.hiveConf = hiveConf;
       this.name = name;
+      this.hiveConf = hiveConf;
```

Comment:
Why was this moved? The order before matched the order the arguments are passed in to the constructor.